clean.cmd

```diff
@@ -0,0 +1,177 @@
+@if "%_echo%" neq "on" echo off
```

Comment:
I'm not sure why this script is not as straight forward as build.cmd. Ideally you would want run.exe to do all of the param parsing. I get how Tools won't be deleted this way, but maybe we should just handle that small special case instead of all of them?